index.js

```diff
@@ -136,61 +136,104 @@ function RpcAgent(opts = {}) {
   this.user = opts.user || 'user';
   this.pass = opts.pass || 'pass';
   this.prot = opts.ssl ? https : http;
+  this.calb = false; // Defaults to promises
```

Comment:
I don't think you need this line. `this.calb` is set every time a command is invoked.